libs/langchain/langchain/document_loaders/tensorflow_datasets.py

```diff
@@ -0,0 +1,79 @@
+from typing import Callable, Dict, Iterator, List, Optional
+
+from langchain.document_loaders.base import BaseLoader
+from langchain.schema import Document
+from langchain.utilities.tensorflow_datasets import TensorflowDatasets
+
+
+class TensorflowDatasetLoader(BaseLoader):
+    """Loads from TensorFlow Datasets into a list of Documents.
+
+    Attributes:
+        dataset_name: the name of the dataset to load
+        split_name: the name of the split to load.
+        load_max_docs: a limit to the number of loaded documents. Defaults to 100.
+        sample_to_document_function: a function that converts a dataset sample
+          into a Document
+
+    Example:
+        .. code-block:: python
+
+            from langchain.document_loaders import TensorflowDatasetLoader
+
+            def mlqaen_example_to_document(example: dict) -> Document:
+                return Document(
+                    page_content=decode_to_str(example["context"]),
+                    metadata={
+                        "id": decode_to_str(example["id"]),
+                        "title": decode_to_str(example["title"]),
+                        "question": decode_to_str(example["question"]),
+                        "answer": decode_to_str(example["answers"]["text"][0]),
+                    },
+                )
+
+            tsds_client = TensorflowDatasetLoader(
+                    dataset_name="mlqa/en",
+                    split_name="test",
+                    load_max_docs=100,
+                    sample_to_document_function=mlqaen_example_to_document,
+                )
+
+    """
+
+    def __init__(
+        self,
+        dataset_name: str,
+        split_name: str,
+        load_max_docs: Optional[int] = 100,
```

Comment:
Missed this first time around --

Best reference I was able to find:  https://peps.python.org/pep-3102/#specification

IMO we should start using `*` as much as possible in the code base for named variables to force caller to invoke by name. It'll force callers code to be easier to read

```suggestion
        *,
        load_max_docs: Optional[int] = 100,
```